templates/sway-predicate-test-rs/template/tests/harness.rs

```diff
@@ -0,0 +1,71 @@
+use fuels::{accounts::{predicate::Predicate}, prelude::*};
+
+// Load abi from json
+abigen!(Predicate(
+    name = "MyPredicate",
+    abi = "out/debug/my-predicate-abi.json"
```

Comment:
Do we also need a PR in docs-hub updating the project names?

Reply:
It will not break the current guides in the docs-hub. Once this PR is merged we can slowly start migrating the existing docs to these new templates.